src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -5751,17 +5773,17 @@ void Fishing_DrawOwner(Actor* thisx, PlayState* play) {
         }
     }
 
-    if ((D_80B7E0AC != 0) && D_80B7A674) {
+    if ((sFishingPlayingState != 0) && sIsRodVisible) {
         Fishing_DrawRod(play);
         Fishing_UpdateLinePos(sReelLinePos);
         Fishing_UpdateLine(play, &sRodTipPos, sReelLinePos, sReelLineRot, sReelLineUnk);
         Fishing_DrawLureAndLine(play, sReelLinePos, sReelLineRot);
 
-        D_80B7A6C4 = input->rel.stick_x;
-        D_80B7A6C8 = input->rel.stick_y;
+        sStickXRelPrev = input->rel.stick_x;
+        sStickYRelPrev = input->rel.stick_y;
```

Comment:
"rel" in `input->rel` stands for "released", because the button fields in that hold the released buttons since the previous frame

but stick_x/y are the "adjusted" values (with a "deadzone" applied), nothing to do with released
so we'd call these variables like `sStickAdjXPrev`, for example elsewhere there is `FileSelectState.stickAdjX` or `PauseContext.stickAdjX`